helm/nessie/README.md

```diff
@@ -76,10 +76,11 @@ $ helm uninstall --namespace nessie-ns nessie
 | cassandra.contactPoints | string | `nil` |  |
 | cassandra.keyspace | string | `"nessie"` |  |
 | cassandra.localDatacenter | string | `nil` |  |
+| dynamodb.profile | string | `"default"` | The name of the profile that should be used, when loading AWS credentials from a profile file. Required only if no secret is provided below. |
```

Comment:
Unrelated to this PR, but should we put this stuff on projectnessie.org?

Reply:
Yes that would be an excellent idea. Currently the docs point users to the codebase:

* https://github.com/projectnessie/nessie/blob/3923cffd1103349457102f5ff458bb1190ed03d5/site/docs/guides/kubernetes.md#L32
* https://github.com/projectnessie/nessie/blob/8157b0b450094fd59492c8b29f2626e7706969f5/grafana/README.md#L5 (3 times in this file, 1 with a wrong anchor)

Reply:
https://github.com/projectnessie/nessie/issues/8388